<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WordPad</class>
 <widget class="QDialog" name="WordPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>-1</width>
    <height>1</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>写字板</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>字体样式</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QDialogButtonBox" name="wp_comfirmButton">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QSpinBox" name="wp_fontSize">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="FontSize">
     <property name="text">
      <string>字体大小</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QFontComboBox" name="wp_fontType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QPlainTextEdit" name="wp_textContent">
     <property name="font">
      <font>
       <family>宋体</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
